sso/resources/org.jaggeryjs.modules.sso/src/main/java/org/jaggeryjs/modules/sso/common/util/Util.java

```diff
@@ -49,6 +50,7 @@
 import javax.xml.parsers.DocumentBuilderFactory;
 import java.io.*;
 import java.security.KeyStore;
+import java.security.NoSuchAlgorithmException;
 import java.util.Random;
 import java.util.zip.*;
```

Comment:
Hi Pamod,
As a result of a security review we had to use SecureRandomIdentifierGenerator to generate the ID. Since it wasn't included earlier in jaggery SSO module, we had to include that.
Thanks
Malintha